### PR TITLE
chore: add debug logs for op consumer controller

### PR DESCRIPTION
**Pull request by lesterli**
This PR adds the debug logs for the OP consumer controller.

```
make lint
```

### CODE REVIEW

File: clientcontroller/opstackl2/consumer.go [hunk 1]
```diff
@@ -182,6 +183,11 @@ func (cc *OPStackL2ConsumerController) SubmitFinalitySig(
 	if err != nil {
 		return nil, err
 	}
+	cc.logger.Debug(
+		"Submit finality signature",
```
Comment (gitferry) [inline clientcontroller/opstackl2/consumer.go:187]:
```suggestion
		"Successfully submitted finality signature",
```

File: clientcontroller/opstackl2/consumer.go [hunk 2]
```diff
@@ -229,7 +235,13 @@ func (cc *OPStackL2ConsumerController) SubmitBatchFinalitySigs(
...
 		return nil, err
 	}
-
+	for _, block := range blocks {
+		cc.logger.Debug(
+			"Submit batch finality signatures",
+			zap.Uint64("height", block.Height),
+			zap.String("block_hash", hex.EncodeToString(block.Hash)),
+		)
+	}
```
Comment (gitferry) [inline clientcontroller/opstackl2/consumer.go:244]: Is it neceesary to iterate all the blocks? Can we do like this:

```
cc.logger.Debug(
	"Successfully submitted finality signatures in a batch",
		zap.Uint64("start_height", blocks[0].Height),
                zap.Uint64("end_height", blocks[len(blocks)-1].Height),
	)
```

**Review by gitferry — APPROVED** [review]

File: clientcontroller/opstackl2/consumer.go (hunk 2 above)
Comment (lesterli) [reply clientcontroller/opstackl2/consumer.go:244]: When debugging today, we needed to check the block hash of fp signs, and thus discovered that the bug is caused by inconsistent hash values. https://github.com/babylonchain/babylon-da-sdk/pull/44/files

Comment (gitferry) [reply clientcontroller/opstackl2/consumer.go:244]: OK then. Let's have the loop

**Pull request merged**